rest_framework_simplify/models.py

```diff
@@ -31,6 +31,17 @@ def __init__(self, *args, **kwargs):
         super(SimplifyModel, self).__init__(*args, **kwargs)
         self.related_items_to_be_saved = []
         self.encrypted_fields = []
+        if hasattr(self, 'change_tracking_fields'):
+            for x in self._meta.local_concrete_fields:
+                if x.name in self.change_tracking_fields:
+                    setattr(self, '_{0}_initial'.format(x.attname), getattr(self, x.attname))
```

Comment:
is attname ever different than x.name?

Reply:
The name would be 'child_one' and the attname would be 'child_one_id'. The idea was that the user wouldn't have to put 'child_one_id' in their change_tracking_fields list. They would put 'child_one' in the list and we would know that we are actually just watching it's 'id'. Do you feel like that would be too confusing?